### PR TITLE
feat: Payload gamemode post

### PR DESCRIPTION
**Pull request by Alystrasz**
##### TODOs

- [x] Add media (images, videos?)
    - [x] Try to record videos in Payload mode using https://www.youtube.com/watch?v=K9bZfsmYcO4
- [x] Add in-detail gamemode description section
- [x] Typos

### CODE REVIEW

**Comment by Alystrasz** [comment]
Also pinging you @Zanieon since you wrote the draft for this blog post.

File: _posts/2024-08-26-payload-gamemode.md [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+other team from pushing a vehicle to its final destination.
+
+This mode has been proven very interesting in both of those games due to the way its main
+objective works and how it creates a specific dynamism in gameplay. However, Titanfall never
+had said gamemode, no cut content or anything of the kind, zero clues in regards to that.
+
+There have been attempts to make this mode in Titanfall; another contributor, named
+[*cat_or_not*](https://github.com/catornot), tried to make it around 2022 but back at that year,
+the navmeshes of the maps in Northstar were very broken and many times, the NPCs would fail to
+path properly, this case preceding my presence as a contributor for the project, as I joined
```
Comment (GeckoEidechse) [inline _posts/2024-08-26-payload-gamemode.md:31]: This sentence is a bit long. Might be worth breaking it up ^^

**Review by GeckoEidechse — COMMENTED** [review]
Awesome thanks for the blog post <3

I marked up whatever I found in regards to typos and phrasing. Some of the sentences are also a bit long and phrased a bit _interesting_ but at the same time that kinda gives the post it's own flair, as between this post and the one's written by you and me you can tell that each person has their own writing style so I actually kinda like it \^\^

**Comment by GeckoEidechse** [comment]
Feel free to ping me when merged so that I can pull the changes on Atlas and get the post published ASAP ^^

File: _posts/2024-08-26-payload-gamemode.md (hunk 1 above)
Comment (Alystrasz) [reply _posts/2024-08-26-payload-gamemode.md]: Addressed in 2b607ba4b78448bb8032d79e16ce0eb1a4700459

File: _data/authors.yml [hunk 2]
```diff
@@ -33,3 +33,17 @@ GeckoEidechse:
...
+  bio      : "Just some level designer around."
+  location : "Troy"
+  email    :
+  links:
+      - label: "Website"
+        icon: "fas fa-fw fa-link"
+        url: "https://www.artstation.com/zanieon"
+      - label: "GitHub"
+        icon: "fab fa-fw fa-github-square"
+        url: "https://github.com/Zanieon"
```
[the remark this answers is not in the document]
Comment (Alystrasz) [reply _data/authors.yml]: Addressed in 0915b467050eb8263985160d3a65a314e49dfdd4

File: _posts/2024-08-26-payload-gamemode.md [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+
+So, due to me dedicating an entire year of my life into making Frontier Defense work, fixing up
+all problems with it and gaining experience in coding content for the game, I finally came into
+realization that I could probably attempt myself at doing such Payload gamemode because I've
+always had interest in playing said mode, despite never playing Team Fortress 2 or Overwatch
+in my life.
+
+Case in point, at a random day, I consulted *cat_or_not* to see how far he went with his own
+
+version of the mode before starting anything on my own — if there was anything I could start
```
Comment (GeckoEidechse) [inline _posts/2024-08-26-payload-gamemode.md:47]: Unintentional line break>
```suggestion
Case in point, at a random day, I consulted *cat_or_not* to see how far he went with his own
version of the mode before starting anything on my own — if there was anything I could start

```

File: _posts/2024-08-26-payload-gamemode.md [hunk 4]
```diff
@@ -0,0 +1,130 @@
...
+from — but the code wasn't touched ever since he lost motivation to work on it due to broken
+navmeshes, it was extremely bare-bones: starting from that code or building an entire new
+one from scratch would be equivalent options.
```
Comment (GeckoEidechse) [inline _posts/2024-08-26-payload-gamemode.md:50]:
```suggestion
navmeshes. It was extremely bare-bones so starting from that code or building an entire new
one from scratch would take equally long.
```

File: _posts/2024-08-26-payload-gamemode.md [hunk 5]
```diff
@@ -0,0 +1,130 @@
...
+normal breaks any person would do (toilet, eat and drink).
+
+That was actually only possible due to the entire work I did on Frontier Defense, as a
+lot of functions and code was adapted from there, alongside some logic from Amped Hardpoint
+gamemode too, with everything working on the basic level the way I conceptualized this mode
+with the touch of Titanfall 2 over it; it even attracted the attention of the Competitive players
+whose only gamemode to play is Capture The Flag.
+
+Pretty much on the week that followed that happening, I did some rebalances and touches
+here and there and bam, there’s this fully brand new game mode that never existed into the
```
Comment (GeckoEidechse) [inline _posts/2024-08-26-payload-gamemode.md:72]: Most of the time in the post it's written as `gamemode` so this should be the same to be consistent ^^
```suggestion
here and there and bam, there’s this fully brand new gamemode that never existed into the
```

File: _posts/2024-08-26-payload-gamemode.md [hunk 6]
```diff
@@ -0,0 +1,130 @@
...
+game actually fully functional now with the way it could play in Titanfall 2’s gameplay terms.
+This was probably my quickest speedrun of coding anything for Northstar project given the 2 day
+timespan, one does not simply drop a whole brand new gamemode that easily, but well, it was
+fun doing that.
+
+## The game mode
```
Comment (GeckoEidechse) [inline _posts/2024-08-26-payload-gamemode.md:78]:
```suggestion
## The gamemode
```

**Review by GeckoEidechse — APPROVED** [review]
Final proof reading. A few more smaller things I found. Should be good to merge after that :D

**Comment by Alystrasz** [comment]
@GeckoEidechse done!

**Pull request merged**